repos/coreclr.proj

```diff
@@ -20,15 +23,21 @@
     <BuildArguments>$(BuildArguments) /bl</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:ILLinkTrimAssembly=false</BuildArguments>
     <BuildArguments>$(BuildArguments) /p:CheckCDefines=false</BuildArguments>
+    <BuildArguments>$(BuildArguments) /p:EnableSourceLink=false</BuildArguments>
 
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments)</BuildCommand>
 
     <BuildCommand Condition="$(Platform.Contains('arm'))">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
 
     <CleanCommand>$(ProjectDirectory)/clean$(ShellExtension)</CleanCommand>
     <OfficialBuildId>20190422.75</OfficialBuildId>
+
+    <OutputPlacementRepoApiImplemented>false</OutputPlacementRepoApiImplemented>
+    <PackagesOutput>$(ProjectDirectory)bin/Product/$(TargetOS).$(Platform).$(Configuration)/.nuget/pkg</PackagesOutput>
   </PropertyGroup>
 
+<Target Name="DebugStuff" AfterTargets="Build"> <Message Importance="High" Text="package output dir: $(PackagesOutput)" /> </Target>
```

Comment:
Is this just for debugging or expected to be permanent?

Reply:
Debugging, removed.